cmd/apps/metallb_app.go

```diff
@@ -0,0 +1,111 @@
+package apps
+
+import (
+	"crypto/rand"
+	b64 "encoding/base64"
+	"fmt"
+	"strings"
+
+	"github.com/alexellis/arkade/pkg/config"
+	"github.com/alexellis/arkade/pkg/k8s"
+	"github.com/alexellis/arkade/pkg/types"
+
+	"github.com/alexellis/arkade/pkg"
+	"github.com/spf13/cobra"
+)
+
+func MakeInstallMetalLB() *cobra.Command {
+	var command = &cobra.Command{
+		Use:          "metallb",
+		Short:        "Install metallb",
+		Long:         `Install metallb for service type:LoadBalancer`,
+		Example:      `arkade install metallb`,
+		SilenceUsage: true,
+	}
+
+	command.Flags().String("address-range", "192.168.0.0/24", "Address range for loadBalancer services")
+
+	command.RunE = func(command *cobra.Command, args []string) error {
+		kubeConfigPath, _ := command.Flags().GetString("kubeconfig")
+		if err := config.SetKubeconfig(kubeConfigPath); err != nil {
+			return err
+		}
+
+		arch := k8s.GetNodeArchitecture()
+		fmt.Printf("Node architecture: %q\n", arch)
+
+		if arch != IntelArch {
+			return fmt.Errorf(OnlyIntelArch)
+		}
+
+		addressRange, _ := command.Flags().GetString("address-range")
```

Comment:
Validate this flag by adding a PreRunE handler, you'll see how in the other apps.